### PR TITLE
feat: add 'min' and 'max' blend modes

### PR DESCRIPTION
**Pull request by dev7355608**
##### Description of change

Added blend modes for min and max blending.

##### Pre-Merge Checklist

- [ ] Tests and/or benchmarks are included
- [ ] Documentation is changed or added
- [ ] Lint process passed (`npm run lint`)
- [ ] Tests passed (`npm run test`)

### CODE REVIEW

**Review by Feu-Secret — APPROVED** [review]

**Comment by bigtimebuddy** [comment]
Could you add a visual render test for this?

**Review by bigtimebuddy — APPROVED** [review]

File: src/rendering/renderers/gl/state/mapWebGLBlendModesToPixi.ts [hunk 1]
```diff
@@ -24,6 +24,25 @@ export function mapWebGLBlendModesToPixi(gl: GlRenderingContext): Record<BLEND_M
     blendMap['screen-npm'] = [gl.SRC_ALPHA, gl.ONE_MINUS_SRC_COLOR, gl.ONE, gl.ONE_MINUS_SRC_ALPHA];
 
     blendMap.erase = [gl.ZERO, gl.ONE_MINUS_SRC_ALPHA];
+
+    const isWebGl2 = 'WebGL2RenderingContext' in globalThis && gl instanceof globalThis.WebGL2RenderingContext;
```
Comment (GoodBoyDigital) [inline src/rendering/renderers/gl/state/mapWebGLBlendModesToPixi.ts:28]: this one might be an issue - if user explicitly specifies WebGL 1 render, this code here assume it WebGL2

**Review by GoodBoyDigital — APPROVED** [review]
lgtm!

File: src/rendering/renderers/gl/state/mapWebGLBlendModesToPixi.ts (hunk 1 above)
Comment (Zyie) [inline src/rendering/renderers/gl/state/mapWebGLBlendModesToPixi.ts:28]: I believe we will want to use the `DOMAdapter` here to make sure this call works on all environments

e.g `const isWebGl2 = !(gl instanceof DOMAdapter.get().getWebGLRenderingContext());`

**Review by Zyie — CHANGES_REQUESTED** [review]

**Review by Zyie — APPROVED** [review]

**Pull request merged**